src/test/java/fi/aalto/cs/apluscourses/integration/ApiTest.java

```diff
@@ -8,8 +8,10 @@
 import io.restassured.http.ContentType;
 import org.apache.http.HttpStatus;
 import org.junit.ClassRule;
+import org.junit.Ignore;
 import org.junit.Test;
 
+@Ignore
 public class ApiTest {
 
   //  For this to work the 'CI=true' environment variable is added to .travis.yml
```

Comment:
Possibly yes. I have no idea at the moment though how difficult it would be. I would assume that the biggest difficulties would arise from setting up the local A+ environment. Here is an alternative option that would at least be really easy:

- Store an API token in GitHub secrets (perhaps one of the test users)
- Test the API directly from a real running instance of A+, such as plus.cs.aalto.fi or minus.cs.aalto.fi.

I think it would actually be quite a nice solution, but I'm not sure if there are any hidden issues with it.